.vscode/dryrun.log

```diff
@@ -0,0 +1,5 @@
+make --dry-run --always-make --keep-going --print-directory
```

Comment:
Same as above.

Can we remove the .vscode files which got uploaded
Please add the following entry in .gitignore -> `.vscode`.